Loadout Cleanup/Revision

John-Dang opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
I removed certain items from Loadout to incentivize taking them from the map, raised the cost on valuable items, and removed plenty of items from the Loadout menu that were never selectable in the first place (space station 13 job specific outfits).
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Makes especially rare clothes, utility items and armors cost more points, removes from the loadout menu heavy immersion breaking items and clothes. It should help to make Yuma Valley a bit more fallout, changed a couple descriptions from Nanotrasen to RobCo so its less immersion breaking. There is a lot more that needs doing but I'm trying to parcel up my work.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Added the checkered suit as an option in Loadout.
del: Removed immersion-breaking items from Loadout and items that were never selectable in the first place.
tweak: Adjusted the loadout cost of utility items, fancy clothes.
balance: Armor items in Loadout now cost much more loadout points, with combat boots removed entirely.
spellcheck: Changed "Nanotrasen" to "RobCo" for a few specific items.
code: changed some code
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

DeveloperLoser commented:
Might want to change the CL, something like

🆑
tweak: Cost of some higher quality clothing.
del: Removed Un-useable/immersion breaking items.
/🆑

John-Dang commented:
> Might want to change the CL, something like
> 
> 🆑
> tweak: Cost of some higher quality clothing.
> del: Removed Un-useable/immersion breaking items.
> /🆑

Thanks, I updated the CL to be actually informative, appreciate the advice!

John-Dang commented:
Closing to try and re-do this without deleting certain .dm files.
